src/core/public/public.api.md

```diff
@@ -778,9 +779,15 @@ export interface NavigateToAppOptions {
     openInNewTab?: boolean;
     path?: string;
     replace?: boolean;
+    skipAppLeave?: boolean;
```

Comment:
Optional flag for plugins to use to opt-out of their registered on app leave behavior.